### PR TITLE
Password input field: show/hide password with icon

### PR DESCRIPTION
**Pull request by SteRiccio**
To show the password show/hide icon inside the input field, I'm using a View that wraps the input field and the icon and giving to the View (almost) the same style as the "normal" input field.

### CODE REVIEW

File: src/arena-mobile-ui/components/PasswordInput/styles.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    alignItems: 'center',
+    padding: 0,
+    paddingVertical: 2,
+  },
+  input: {
+    width: '95%',
+  },
+});
+
+export default styles;
```
Comment (ramirobg94) [inline src/arena-mobile-ui/components/PasswordInput/styles.js:22]: Maybe you can export as a component into the Input

export { default as Input, PasswordInput } ...

And compose the styles, ={[style.1, style.2]}
Also I prefer not to use %  in styles

File: src/screens/ConnectionSettings/screen.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ import Card from 'arena-mobile-ui/components/Card';
 import Header from 'arena-mobile-ui/components/Header';
 import Input from 'arena-mobile-ui/components/Input';
```
Comment (ramirobg94) [inline src/screens/ConnectionSettings/screen.js:15]:
```suggestion
import Input, { PasswordInput } from 'arena-mobile-ui/components/Input';
```

File: src/arena-mobile-ui/components/PasswordInput/styles.js (hunk 1 above)
Comment (SteRiccio) [reply src/arena-mobile-ui/components/PasswordInput/styles.js:22]: maybe it's better to keep them in separate files: Input could go to a Input.js file, PasswordInput to its own PasswordInput.js file and they could be both exported by an index.js file in Input. If I move it inside index.js, the style.js should be renamed and put in the same folder... a bit messy...

File: src/screens/ConnectionSettings/screen.js (hunk 2 above)
Comment (SteRiccio) [reply src/screens/ConnectionSettings/screen.js:15]: Why don't we export all the components from an index.js file in the components folder?

File: src/arena-mobile-ui/components/PasswordInput/styles.js (hunk 1 above)
Comment (SteRiccio) [reply src/arena-mobile-ui/components/PasswordInput/styles.js:22]: composing styles is nice, but even the spread operator is allowed, and I think that the component should not be aware of how the style of the component is made up, so having it all in the styles.js file is better for me...

**Pull request merged**